### DISCUSSION TITLE
Hard limit on Max. Characters per line

### DISCUSSION THREAD

**Discussion by Estevarena** [SubtitleEdit/subtitleedit#7580] [Ideas]
Hi, everyone! I was wondering if it would be possible to add an optional limit to the maximum number of characters per line, that automatically moves the cursor to the next line, and maybe prevents you from writing on, when you reach the end of the subtitle.
Like MS Word and LO Writer do when you reach the margin of the page.
Maybe that already exists, but I couldn't find it.
Thanks!

**Comment by Flitskikker** [comment]
Have you tried Settings > General > Auto-wrap while typing?

**Reply by Estevarena** [reply]
I'm actually reopening this, because that option seems to work only in the first line, but not on the following. Also, it would be nice to have these limits work for the overall subtitle. If the Single line max. length is set to 42, and the Max. number of lines is set to 2, it shouldn't allow the user to create a third line. Maybe pressing Intro should create a new caption? I understand this may not be ideal for many, that's why I'm suggesting this to be implemented as an option.

**Comment by Estevarena** [comment]
I overlooked that... Thanks!!
